joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -1089,8 +1093,7 @@ class AstCreator(filename: String, global: Global)
     val column         = localIdentifier.getSymbol().getCharPositionInLine()
     val line           = localIdentifier.getSymbol().getLine()
     val name           = getActualMethodName(localIdentifier.getText)
-    val methodFullName = s"$filename:$name"
-
+    val methodFullName = packageContext.packageTable.getMethodFullNameUsingName(packageStack.toList, code).head
```

Comment:
Use `name` and not `code` to lookup for the methodfullname
The first argument to this method should be the classname as we discussed and not the package name. It will be used when type recovery is done. For now, nothing will be passed.

Reply:
The class stack cannot be considered the package.I would suggest passing only the method name argument till we figure out what should be the exact content of the class

Reply:
@rahul-privado Yes yes, this `packageStack` is only for optimisation (use in narrowing down the searching methodName in imports defined in the file). we still matching the method by its name.